### ISSUE TITLE
Does not support "StepDefinition" annotation 

### ISSUE THREAD

**Issue by iamfreid** [SpecFlowOSS/resharper-specflow#17]
Hi!

**This one doesn't work:**
`Given Some step`
```
[StepDefinition(@"Some step")]
        public void GivenSomeStepCommon()
        {
            ScenarioContext.StepIsPending();
        }
```

**But it works:**
`Given Some step`
```
[Given(@"Some step")]
        public void GivenSomeStepCommon()
        {
            ScenarioContext.StepIsPending();
        }
```

**Comment by Socolin** [comment]
Hello,

Thanks for reporting the problem, I've fixed this in 1.3.2, available [here](https://github.com/SpecFlowOSS/resharper-specflow/releases/tag/v1.3.2) or on the marketplace soon

**Comment by iamfreid** [comment]
Thank you! It works.
